core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -15,3 +15,105 @@
 /**
 * @fileoverview End-to-end tests for user profile features.
 */
+
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+
+var LearnerDashboardPage =
+  require('../protractor_utils/LearnerDashboardPage.js');
+var LibraryPage = require('../protractor_utils/LibraryPage.js');
+var SubscriptionDashboardPage =
+  require('../protractor_utils/SubscriptionDashboardPage.js');
+
+describe('Profile Features functionality', function() {
+  var libraryPage = null;
+  var learnerDashboardPage = null;
+  var subscriptionDashboardPage = null;
+
+  var creator1Id = 'FirstCreator';
+  var creator2Id = 'SecondCreator';
+  users.createUser('FirstUser@profileFeatures.com',
+    'FirstUserprofileFeatures');
+  users.createUser('SecondUser@profileFeatures.com',
+    'SecondUserprofileFeatures');
+  users.createUser(creator1Id + '@profileFeatures.com',
+    creator1Id);
+  users.createUser(creator2Id + '@profileFeatures.com',
```

Comment:
I think the creation of users should be put in the beforeAll block.

Reply:
@apoorv1997 is this comment resolved?